YANG/ccamp/client-pm/ietf-service-pm.yang

```diff
@@ -118,49 +118,60 @@ module ietf-service-pm {
     description
       "Base type of the performance parameter being monitored.";
   }
-  
+
   identity near-frame-loss {
     base performance-parameter-type;
     description
-      "Near frame loss, using one-way eth loss measure, 
+      "Near frame loss, using one-way eth loss measure,
        the sampling point is the MEP.";
   }
-  
+
   identity far-frame-loss {
     base performance-parameter-type;
     description
-      "Far frame loss, using one-way eth loss measure, 
+      "Far frame loss, using one-way eth loss measure,
        the sampling point is the MEP.";
   }
-  
+
   identity one-way-delay {
     base performance-parameter-type;
     description
       "One way delay.";
   }
-  
+
   identity two-way-delay {
     base performance-parameter-type;
     description
       "Two way delay.";
   }
-  
+
   identity receive-packets {
     base performance-parameter-type;
     description
       "Total number of received packets.";
   }
-  
+
   identity transmit-packets {
     base performance-parameter-type;
     description
       "Total number of transmitted packets.";
   }
 
+  identity ingress-bandwidth {
+    base performance-parameter-type;
+    description
+      "Current bandwidth usage of the ingress traffic.";
+  }
+
+  identity egress-bandwidth {
+    base performance-parameter-type;
+    description
+      "Current bandwidth usage of the egress traffic.";
+  }
```

Comment:
The definition is not clear to me

The "bandwidth usage" should be a percentage: is it the percentage of the bandwidth used by the service against the bandwidth allocated to the service or against the bandwidth of the access link?

For example, if the customer is sending 1Gb/s of traffic for a 2Gb/s service over a 10Gb/s access link: is the ingress-bandwidth 50% (1/2) or 10% (1/10)?

In the former case, is the allocated bandwidth the CIR or something else?

How is the bandwidth used by the service calculated?

Reply:
@agva123 , @haomianzheng , @italobusi : clarified that the performance parameter is the rate of the ingress/egress traffic

```suggestion
  identity ingress-rate {
    base performance-parameter-type;
    description
      "The rate of the ingress traffic.";
  }

  identity egress-rate {
    base performance-parameter-type;
    description
      "The rate of the egress traffic.";
  }
```

Further work is required to:
- How the rate for packet traffic is measured: issue #88 
- At which point service PM is measured: issue #89